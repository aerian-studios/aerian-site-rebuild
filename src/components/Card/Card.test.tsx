/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { Card } from "./index";

describe("Card", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(<Card className="myClass" link="/my-bogus-site" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
