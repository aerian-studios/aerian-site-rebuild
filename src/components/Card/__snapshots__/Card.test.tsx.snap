// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className="   fade"
>
  <div>
    <p>
      Hi
    </p>
  </div>
</div>
`;

exports[`Card renders visible cards 1`] = `
<div
  className="   "
>
  <div>
    <p>
      Hi
    </p>
  </div>
</div>
`;
